Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -184,6 +184,40 @@
             "helpMarkDown": "Select the option to enable msdeploy flag MSDEPLOY_RENAME_FILES_FLAG in Azure App Service application settings. The option if set enables msdeploy to rename locked files that are locked during app deployment"
         },
         {
+            "name": "ScriptType",
+            "type": "pickList",
+            "label": "Script Type",
+            "defaultValue": "",
+            "options": {
+                "": "Select Script Type",
+                "Inline Script": "Inline Script",
+                "File Path": "File Path"
+            },
+            "groupName": "AdditionalDeploymentOptions",
+            "helpMarkDown": "Type of the script: File Path or Inline Script"
```

Comment:
Help text should be clearly calling out whether the custom script should be locally available or on target.